src/components/CreatePostInput/index.js

```diff
@@ -37,7 +37,7 @@ const CreatePostInput = ({
     useHastagMention('');
 
   React.useEffect(() => {
-    if (new Date().valueOf() > 0) updateHashtag(message, topics, setTopics);
+    if (new Date().valueOf() > 0) updateHashtag(message, topics);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The condition `new Date().valueOf() > 0` will always be true because `Date().valueOf()` returns the current timestamp. If this is intended as a placeholder for future conditions, consider adding a comment to clarify. Otherwise, it's unnecessary and can be removed.
```diff
-    if (new Date().valueOf() > 0) updateHashtag(message, topics);
+    updateHashtag(message, topics);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->